src/Orleans/Configuration/ConfigurationExtensions.cs

```diff
@@ -57,5 +57,22 @@ public static class ConfigurationExtensions
 
             return properties;
         }
+
+
+        /// <summary>
+        /// Configures all cluster nodes to use the specified startup class for dependency injection.
+        /// </summary>
+        /// <typeparam name="TStartup">Startup type</typeparam>
+        public static void UseStartupClass<TStartup>(this ClusterConfiguration config) 
```

Comment:
I'd rename it to UseStartupType, to be consistent with the actual configuration setting.